### ISSUE TITLE
6.18.1 for iOS and Android

### ISSUE THREAD

**Issue by giovannibedetti** [csound/csound#1753]
In the 6.18.1 release there are no Android and iOS zip files.
Can they be added?

The issue I have is that I noticed that the ftgenonce opcode is missing in release 6.18, I tested on Windows and Android. Some others could be missing too. 
For context, [here is the list of opcodes](https://www.diffchecker.com/mk8I50i0/) obtained with the command: "csound -z" for version 6.18.1 vs 6.18.

**Comment by vlazzarini** [comment]
I don't think there are any differences in 6.18.1 and 6.18 for iOS. Since iOS does not have plugins, ftgetonce will not be there (it is in a plugin).

If it's missing on Windows, we need to ask @stekyne

**Comment by giovannibedetti** [comment]
Oh ok of course! I could have noticed (I had the same issue in the past with the alwayson opcode).
Not sure why until I installed 6.18.1 I wasn't able to use plugin opcodes on Windows. I think I could have removed them from my system some time ago for testing purposes.
About the releases, I guess I don't need to update Csound libraries (Windows, macOS, Android and iOS) with this release on CsoundUnity, since it's basically unaffected by the fixes above.
Sorry for the false flag.
Closing!
